code/development-env/scripts/helm-install-datalabs-locally.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Creates necessary secrets within the target kubernetes cluster such that a helm install
+# of the datalabs chart can be installed (chart available in datalab-k8s-manifests repo).
+# This script will also carry out the helm install of the datalab chart.
+
+# The domain you are running datalabs on locally. Either datalabs.internal or datalabs.localhost.
+targetDomain="datalabs.internal"
+# The namespace to install the helm chart and the datalab services to
+targetNamespace="devtest"
+mongoPassword="datalabs-root-p"
+
+# The path to the datalab helm chart to install. You will likely need to have a local
+# of the datalabs-k8s-manifests repo to have this chart available for install.
+helmChartPath="<path to datalab helm chart>"
```

Comment:
Would it make sense to have this script in the datalabs-k8s-manifests repo?

Reply:
The script also points to the self signed certificates that live in the development env directory. Based on it needing these certificates and the use case for the script (likely people who are developing datalabs) I thought it was a slightly better fit where it is. If you think otherwise, I would be happy to move it.

Reply:
That makes sense then.  However, I expect most people will have the datalab repo cloned adjacently to the datalab-k8s-manifests repo - perhaps you could do a relative path based that assumption and a comment, rather than always having to modify the script?